### DISCUSSION TITLE
Ability to switch Entry Condition to "or" rather than "and"

### DISCUSSION THREAD

**Discussion by turnstylerj** [craftcms/cms#11542] [Ideas] [answered]
Currently, when multiple conditions are added to the Entry Condition selection, they stack with "and" behavior (i.e. Related to category x _and_ y). In other systems (like Advanced Custom Fields in WordPress), it's possible to switch this from "and" to "or" (and is clearer which behavior is currently active). In Craft, this might look like:

![Screen Shot 2022-07-01 at 1 49 21 PM](https://user-images.githubusercontent.com/42223501/176966246-022e8687-1095-424f-b6ca-05185af8c017.png)
![Screen Shot 2022-07-01 at 1 49 31 PM](https://user-images.githubusercontent.com/42223501/176966243-bb4cd0db-506f-4316-8fca-c591c9465b29.png)

This would be especially useful with category relations (i.e. Show this field if the entry is related to category x _or_ y). Possible to add this to Craft 4?

**Comment by brandonkelly** [comment, marked as answer]
This was previously requested by #11287 :)
